File: src/main/scala/my/spark/examples/scala/LogCount.scala
package my.spark.examples.scala

import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._

object LogCount {
  def main(args: Array[String]) {

    if (args.length < 1) {
      System.err.println("Usage: LogCount <directory|file>")
      System.exit(1)
    }
    val sc = new SparkContext()
    val count = sc.textFile(args(0)).count()
    println( "Log Count: " + count)
    sc.stop()
  }
}

File: src/main/python/LogCount.py
import sys
from pyspark import SparkContext

if __name__ == "__main__":
  if len(sys.argv) < 2:
    print >> sys.stderr, "Usage: LogCount <directory|file>"
    exit(-1)
  sc = SparkContext()
  count = sc.textFile(sys.argv[1]).count()
  print "Log Count: ", count
